lib/hammer_cli/settings.rb

```diff
@@ -34,10 +34,14 @@ def self.load_from_paths(files)
 
     def self.load_from_file(file_path)
       if File.file? file_path
-        config = YAML::load(File.open(file_path))
-        if config
-          load(config)
-          path_history << file_path
+        begin
+          config = YAML::load(File.open(file_path))
+          if config
+            load(config)
+            path_history << file_path
+          end
+        rescue Exception => e
+          warn _("Couldn't load configuration file %{path}: %{message}") % { path: file_path, message: e.message }
```

Comment:
Almost perfect, could you please prefix the message  with "Warning: " as we do elsewhere.